src/analysis/smooth.jl

```diff
@@ -1,20 +1,32 @@
 # From StatsMakie
 function _linear(x::AbstractVector{T}, y::AbstractVector;
-                 n_points = 100, interval = :confidence) where T
+                 n_points = 100, wts=T[],
```

Comment:
Maybe `wts = similar(x, 0)`, to be in line with how GLM does it? Note also the space around `=` in keyword arguments.